controls/SV-204631.rb

```diff
@@ -19,20 +19,19 @@
     # yum list installed pam_pkcs11
     pam_pkcs11-0.6.2-14.el7.noarch.rpm
     If the "pam_pkcs11" package is not installed, this is a finding.'
-  desc 'fix', 'Configure the operating system to implement multifactor authentication by
-installing the required packages.
+  desc 'fix', 'Configure the operating system to implement multifactor authentication by installing the required packages.
 
-Install the "esc" and "pam_pkcs11" packages on the system with the
-following command:
+Install the pam_pkcs11 package with the following command:
 
-# yum install esc pam_pkcs11'
+# yum install pam_pkcs11'
```

Comment:
V3R6 required `esc` and `pam_pkcs11` and V3R10 requires only `pam_pkcs11`. Not sure why all of these packages are set as a default requirement for MFA:
```yml
  # V-204631
  - name: mfa_pkg_list
    description: The list of packages needed for MFA on RHEL
    type: Array
    value:
      - "nss-tools"
      - "nss-pam-ldapd"
      - "esc"
      - "pam_pkcs11"
      - "pam_krb5"
      - "opensc"
      - "pcsc-lite-ccid"
      - "gdm"
      - "authconfig"
      - "authconfig-gtk"
      - "krb5-libs"
      - "krb5-workstation"
      - "krb5-pkinit"
      - "pcsc-lite"
      - "pcsc-lite-libs"
```

Also, should the following code block be a conditional fail (instead of a skip) if multifactor authentication packages are not installed?
```rb
if smart_card_status.eql?('disabled')
      impact 0.5
      describe 'The system is not smartcard enabled thus this control is Not Applicable' do
        skip 'The system is not using Smartcards / PIVs to fulfil the MFA requirement, this control is Not Applicable.'
      end
```

I can think of one scenario where it could be a skip:
```rb
# pseudocode
if smart_card_status.eql?('disabled')
  describe package('pam_pkcs11') do
    it { should be_installed }
  end
else
  impact 0.0
  describe 'Manually review and verify that hardware MFA solutions are used as stated in the inputs' do
    skip 'Manually review and verify that Smart Cards/ PIVs/ CACs/ Tokens are used as stated in the inputs'
  end
end
```

Reply:
Other controls impacted by similar logic:

- [ ] SV-204441

Reply:
After discussing with @aaronlippold, we decided to add verbiage about the dependencies for `pam_pkcs11`

Reply:
Verbiage added and no other changes needed